docs/api/index.rst

```diff
@@ -8,13 +8,13 @@ describing what they are and what they do. The APIs are grouped into the followi
 
    .. card::
       :title: Prediction Tasks
-      :link: autogluon.task.html
+      :link: autogluon.core.task.html
```

Comment:
documentation issue

Reply:
The docs work, though `autogluon.core.task` doesn't make sense because it's fractured into separate sub-modules: `autogluon.vision.task`, etc. I think we can refactor this in the follow-up PR. I'm currently trying to minimize any other changes beyond moving and necessary fixes to keep the code running.